pages/docs/concepts/asyncapi-document/add-server.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Add servers
+weight: 295
+---
+
+A server represents a message broker handling communication between producers and consumers. Adding and defining servers is useful because it specifies where and how to connect. The connection facilitates where to send and receive messages.
+
+A unique key identifies the server and contains information about the server's connection details, such as the URL, protocol, and authentication. The diagram below describes how to define and add servers in AsyncAPI.
+
+```mermaid
```

Comment:
Hey @kakabisht, did you remember to review your work before requesting a review? Notice the diagram issue where the arrow overlaps into the next subgraph.  

<img width="1453" alt="Screenshot 2023-11-16 at 5 05 39 PM" src="https://github.com/asyncapi/website/assets/19964402/db96c477-d4d9-4bc1-90bb-e1472f13928e">